salt/modules/virt.py

```diff
@@ -2578,6 +2791,7 @@ def update(
     graphics=None,
     live=True,
     boot=None,
+    numatune=None,
```

Comment:
this kwarg should be after `boot_dev` to safely cover the case for someone who is calling these kwargs as positional parameters.

Reply:
That's right ! Thanks Megan.